test/functional/helpers.lua

```diff
@@ -719,6 +724,19 @@ function module.pending_win32(pending_fn)
   end
 end
 
+-- Helper to skip tests. Returns true in FreeBSD systems.
+-- pending_fn is pending() from busted
+function module.pending_freebsd(pending_fn)
```

Comment:
we need to try to avoid introducing more of this kind of thing. we have `skip_fragile` already. We could use that, and define `TEST_SKIP_FRAGILE` on sourcehut :(

https://github.com/neovim/neovim/blob/ac73f75fc0073db3e0bb1c0c2758b3dc34a3ed51/test/functional/helpers.lua#L737